scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
@dnephin Yes, I've tried it and the editor workflow is much better than wasting energy running unit tests that don't need to be run. More often than not, the code I am writing doesn't compile until it works.

But the point is not to have arguments about the "correct" development workflow. You can do what works for you, but don't push those into the project itself.